src/test/java/io/github/ititus/math/number/BigComplexTests.java

```diff
@@ -0,0 +1,105 @@
+package io.github.ititus.math.number;
+
+import org.assertj.core.data.Offset;
+import org.junit.jupiter.api.Test;
+
+import static io.github.ititus.assertions.Assertions.assertThat;
+import static io.github.ititus.math.number.BigComplexConstants.ONE;
+import static io.github.ititus.math.number.BigComplexConstants.ZERO;
+import static io.github.ititus.math.number.BigRational.of;
+import static io.github.ititus.math.number.BigRationalConstants.*;
+import static org.assertj.core.api.Assertions.assertThatExceptionOfType;
+import static org.assertj.core.data.Offset.strictOffset;
+
+public class BigComplexTests {
+    public static final BigComplex ONE_I_TWO = BigComplex.of(BigRationalConstants.ONE, TWO);
+    public static final BigComplex TWO_I_TWO = BigComplex.of(TWO, TWO);
+    public static final BigRational TWICE_TWO_SQRT = TWO.sqrt().multiply(TWO);
+    public static final Offset<BigRational> EPSILON = strictOffset(of("0.001"));
+    public static final Offset<BigRational> STRICT_EPS = strictOffset(of("0.000001"));
```

Comment:
```suggestion

    private static final BigRational TWICE_TWO_SQRT = TWO.multiply(TWO.sqrt());

    private static final BigComplex ONE_I_TWO = BigComplex.of(BigRationalConstants.ONE, TWO);
    private static final BigComplex TWO_I_TWO = BigComplex.of(TWO, TWO);
    
    private static final Offset<BigRational> EPSILON = strictOffset(of("0.001"));
    private static final Offset<BigRational> STRICT_EPS = strictOffset(of("0.000001"));
```

Reply:
Please use only this offset:
```java
private static final Offset<BigRational> EPS = strictOffset(ofExp(1, -32));
```
with some big number. `-32` ist just a suggestion.